src/features/user/TreeMapper/components/InterventionPage.tsx

```diff
@@ -387,20 +383,20 @@ export function LocationDetails({
   );
 }
 
-export default function PlantLocationPage({
+export default function InterventionPage({
   location,
   setSelectedLocation,
-  plantLocations,
+  intervention,
 }: Props): ReactElement {
   const router = useRouter();
 
   const handleBackButton = () => {
     if (location?.type === 'sample-tree-registration') {
-      for (const iKey in plantLocations) {
-        const i = iKey as keyof typeof plantLocations;
-        if (Object.prototype.hasOwnProperty.call(plantLocations, i)) {
-          const pl = plantLocations[i] as PlantLocation;
-          if (pl.id === (location as SamplePlantLocation)?.parent) {
+      for (const iKey in intervention) {
+        const i = iKey as keyof typeof intervention;
+        if (Object.prototype.hasOwnProperty.call(intervention, i)) {
+          const pl = intervention[i] as Intervention;
+          if (pl.id === (location as SampleIntervention)?.parent) {
```

Comment:
The use of `pl` should be avoided, as it was short form for `plantLocation` and lacks context in the updated code.